srcpkgs/libcaca/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'libcaca'
 pkgname=libcaca
 version=0.99.beta19
-revision=8
+revision=9
 build_style=gnu-configure
 hostmakedepends="libtool automake pkg-config"
+makedepends="ncurses-devel imlib2-devel $(vopt_if x11 'libX11-devel')
+ $(vopt_if opengl 'libfreeglut-devel')"
```

Comment:
This makes it enable the `libX11` backend as well, because `imlib2-devel` depends on `libX11-devel` :p 

You should add some configure args.

(also ideally shorten the commit title a bit - the message is fine. maybe move the opengl stuff to the message)

Reply:
My bad, didn't check the dependencies. I added a `vopt_if` for `configure_args`.

`cacaview` now open images within the terminal so I think it works.